Common/kvstore/kvstore.c

```diff
@@ -77,6 +77,36 @@ static size_t xReadEntryOrDefault( KVStoreKey_t xKey,
         xLength = kvStoreDefaults[ xKey ].length;
     }
 
+/* TEST_AUTOMATION_INTEGRATION is set in ota_config.h, help us to set attributes easily. */
```

Comment:
Is there still no way to configure this after flashing the device?

Reply:
We may be able to set it manually by script, such as provision.py, but we need a automatic method to run IDT.
IDT supports modify test_param_config.h & test_execution_config.h for every test cases.
If we want to run IDT, we can't configure it by script.

Thanks.